input.go

```diff
@@ -43,6 +43,17 @@ type AwsIntegrationInput struct {
 	AwsTagsOverrideOwnership *bool     `json:"awsTagsOverrideOwnership,omitempty" yaml:"awsTagsOverrideOwnership,omitempty" example:"false"`    // Allow tags imported from AWS to override ownership set in OpsLevel directly. (Optional.)
 }
 
+// AzureResourcesIntegrationInput specifies the input fields used to create and update an Azure resources integration.
+type AzureResourcesIntegrationInput struct {
+	Name                  *string   `json:"name,omitempty" yaml:"name,omitempty" example:"example_name"`                                        // The name of the integration. (Optional.)
+	OwnershipTagKeys      *[]string `json:"ownershipTagKeys,omitempty" yaml:"ownershipTagKeys,omitempty" example:"['tag_key1', 'tag_key2']"`    // An array of tag keys used to associate ownership from an integration. Max 5. (Optional.)
```

Comment:
Just noticing this now but this is incorret.  The schema spec of this is `ownershipTagKeys: [String!]`. notice how on the inside of the array the non-null is on the string - not the array itself.  This means it should be

```suggestion
	OwnershipTagKeys      []*string `json:"ownershipTagKeys,omitempty" yaml:"ownershipTagKeys,omitempty" example:"['tag_key1', 'tag_key2']"`    // An array of tag keys used to associate ownership from an integration. Max 5. (Optional.)
```